devices/HapticGlove/src/HapticGlove.cpp

```diff
@@ -537,22 +551,24 @@ class HapticGlove::SenseGloveImpl::SenseGloveHapticActuator : public wearable::a
 
     bool setHapticCommand(double& value) const override
     {
+        return true;
```

Comment:
Why this returns ``true`` if it does nothing? If it is not necessary anymore, consider removing it, or returning an error message

Reply:
yes, you are right. These methods are no longer used. I will remove them.